src/pages/cli/graphql/overview.mdx

```diff
@@ -259,6 +259,49 @@ Amplify.API.mutate(ModelMutation.create(todo),
 
 The result output should indicate to you that the record was successfully deleted!
 
+## Update schema
+
+If you want to update your API, open your project's `backend/api/~apiname~/schema.graphql` file (NOT the one in the `backend/api/~apiname~/build` folder) and edit it in your favorite code editor. You can compile the `backend/api/~apiname~/schema.graphql` by running:
```

Comment:
```suggestion
If you want to update your API, open your project's `amplify/backend/api/<api-name>/schema.graphql` file (NOT the one in the `amplify/backend/api/<api-name>/build` folder) and edit it in your favorite code editor. You can compile the `amplify/backend/api/<api-name>/schema.graphql` file by running:
```

Reply:
^ this is minor but since we're here https://github.com/aws-amplify/docs/issues/4137